src/interfaces/operation.ts

```diff
@@ -309,18 +309,20 @@ export interface AssociatedItem extends DataItem {
 }
 
 /**
- * Дескриптор включения операции на виджете:
- * - либо строка (если надо просто включить/исключить операцию или группы)
- * - либо объект, если это группа в которой нужно выборочно включить или исключить операцию
+ * Descriptor enabling operation on widget:
+ * - string (if you just need to include / exclude operation or groups)
+ * - object, if this is group in which you want to selectively include or exclude the operation
  * 
- * @param type Тип операции; строка, уникально идентифицирующая операцию на виджете
- * @param include Список включаемых операций
- * @param exclude Список исключаемых операций
+ * @param type Type of transaction; a string that uniquely identifies the operation on the widget
+ * @param include List of included operations or groups operations
+ * @param exclude List of excluded operations or groups operations
+ * @param defaultSave default no crud save action
  */
 export type OperationInclusionDescriptor = string | {
     type: OperationType,
     include?: OperationInclusionDescriptor[],
-    exclude?: OperationType[]
+    exclude?: OperationType[],
+    defaultSave?: string
```

Comment:
I don't think it is a correct place to configure operations:
* Type name is misleading (`OperationInclusionDescriptor` gives no hint that it also handles autosaves)
* `OperationInclusionDescriptor` is used for optional `include` config:
https://github.com/tesler-platform/tesler-ui/blob/1c94589c7d9c1ff06a5220f775302e44f97a9956/src/interfaces/widget.ts#L303-L311
It means that if you want to specify autosave action you also should configure widget operations via `include` property and autosave can't be used if your operations configured with `exclude` or does not care about inclusion/exclusion logic at all

`WidgetOperations` should be used for specifying default save action instead.

Reply:
removed